components/ModEditSubmissionForm.vue

```diff
@@ -1,4 +1,23 @@
 <template>
+    <div
+        v-show="modalStore.isOpen"
+        class="fixed top-0 left-0 flex items-center justify-center h-full w-full z-10 bg-secondary bg-opacity-40"
+    >
+        <Modal
+
+            class=" min-h-20 min-w-20"
+        >
+            <div class="flex flex-col aspect-square h-96 items-center justify-around bg-primary-inverted p-10 rounded">
+                <span class="font-bold text-3xl">{{ $t('modSubmissionForm.confirmationModal') }}</span>
```

Comment:
Nice! I would add these changes to the first commit and skip checking in the raw text changes.